testsuite/MDAnalysisTests/coordinates/test_rdkit.py

```diff
@@ -167,11 +167,21 @@ def peptide(self):
         u.add_TopologyAttr('elements', elements)
         return u.select_atoms("resid 2-12")
 
+    @pytest.fixture
+    def uo2(self):
+        return mda.Universe.from_smiles("O=O")
+
+    def test_no_atoms_attr(self):
+        rdkit_converter = mda._CONVERTERS["RDKIT"]().convert
+        with pytest.raises(TypeError, match=""):
+            rdkit_converter("foo")
+
     @pytest.mark.parametrize("smi", ["[H]", "C", "O", "[He]"])
     def test_single_atom_mol(self, smi):
         u = mda.Universe.from_smiles(smi, addHs=False,
                                      generate_coordinates=False)
-        mol = u.atoms.convert_to("RDKIT")
```

Comment:
Sorry, I think I'm just being silly and forgetting a very obvious thing. Could you remind me why these are all being switched away from `convert_to`?

Reply:
`convert_to` doesn't pass arguments to the underlying converter, it was in a PR at some point though (#2882 )

Reply:
Doesn't this behaviour contradict the docstring? I.e. ":func:`set_converter_cache_size`. However, ``ag.convert_to("RDKIT")``
    followed by ``ag.convert_to("RDKIT", NoImplicit=False)`` will not use the"

Or was the argument that we would merge #2882 before this PR?

Reply:
The point is the converter modules weren't really documented to be instantiated like `c = mda.coordinates.RDKit.RDKitConverter(); c.convert(...)` but usually go through the `convert_to` AtomGroup method.
So yeah I assumed 2882 would be merged before v2.0 comes out

Reply:
alright, let's see if we can revive #2882 then